code/src/components/app-table/components/app-table-box/AppTableBox.types.ts

```diff
@@ -0,0 +1,3 @@
+import { TableProps } from "@mui/material/Table";
+
+export type AppTableBoxProps = TableProps;
```

Comment:
just import this type inside `AppTableBox `. It is not a problem import this type there. To avoid re-assigning type